### PR TITLE
Notebooks documentation

### PR DESCRIPTION
**Pull request by enolfc**
# Summary 

----

**Related issue :**

### CODE REVIEW

**Comment by enolfc** [comment]
Just taken from the existing docs in rtfd.io

File: content/users/Notebooks/_index.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+title: "Notebooks"
+linkTitle: "Notebooks"
+type: docs
+description: "Documentation related to EGI Notebooks"
+weight: 40
+---
+
```
Comment (gwarf) [inline content/users/Notebooks/_index.md]:
```suggestion
```

**Comment by enolfc** [comment]
> I've just made a lightening fast review, assuming it was already reviewed earlier. So only one small suggestion.

I did a bit of reorganisation, but content was indeed reviewed in previous PR for the other repo

> We may want to improve the automated checks to be sure we do all things in a coherent way (when there are various ways to write markdown or to use hugo short codes and so on).

I would expect mdl does force some coherent markdown... for the shortcodes, we should at least take a decision on how to include images and we should review relative links

**Comment by gwarf** [comment]
> for the shortcodes, we should at least take a decision on how to include images and we should review relative links

I think we will need a clear contribution and style guide at some point in time...

**Pull request merged**